react.js

```diff
@@ -28,29 +71,25 @@ module.exports = {
     'react/jsx-indent': [2, 2],
     // Validate JSX has key prop when in array or iterator
     'react/jsx-key': 2,
+    // Limit maximum of props on a single line in JSX
+    'react/jsx-max-props-per-line': 0,
     // Prevent usage of .bind() and arrow functions in JSX props
     'react/jsx-no-bind': 0,
     // Prevent duplicate props in JSX
     'react/jsx-no-duplicate-props': 2,
+    // Prevent usage of unwrapped JSX strings
+    'react/jsx-no-literals': 0,
     // Disallow undeclared variables in JSX
     'react/jsx-no-undef': 2,
     // Enforce PascalCase for user-defined JSX components
     'react/jsx-pascal-case': 2,
+    // Enforce props alphabetical sorting
+    'react/jsx-sort-props': 0,
+    // Enforce spaces before the closing bracket of self-closing JSX elements
+    'react/jsx-space-before-closing': [2, 'always'],
```

Comment:
I like this style, but there are going to be a lot of violations in existing code - do you plan to fix them all?